src/models/ui.ts

```diff
@@ -95,6 +95,7 @@ export type UiState = {
   resetLayout: boolean;
   clusterPaneIconHighlighted: boolean;
   clusterStatusHidden: boolean;
+  reloadFileTree: boolean;
```

Comment:
hm... why is this necessary? doesn't the file-tree react to changes in the fileMap ?

Reply:
It does - but there is another problem I added this as temporary